wasm/sdk.go

```diff
@@ -0,0 +1,1178 @@
+				})
+			}
+
+			resolve.Invoke(map[string]interface{}{
+				"result": string(readPool),
+			})
+		}()
+
+		return nil
+	})
+
+	promiseConstructor := js.Global().Get("Promise")
+	return promiseConstructor.New(handler)
+}
+
+// // ReadPoolLock locks given number of tokes for given duration in read pool.
+func ReadPoolLock(this js.Value, p []js.Value) interface{} {
+	dur, _ := time.ParseDuration(p[0].String()) // time.Duration,
+	allocID := p[1].String()
+	blobberID := p[2].String()
+	tokens, _ := strconv.ParseInt(p[3].String(), 10, 64)
+	fee, _ := strconv.ParseInt(p[4].String(), 10, 64)
+
+	handler := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		resolve := args[0]
+		reject := args[1]
+
+		go func() {
+			err := sdk.ReadPoolLock(dur, allocID, blobberID, tokens, fee)
+			if err != nil {
+				reject.Invoke(map[string]interface{}{
+					"error": fmt.Sprintf("ReadPoolLock failed. Reason: %s", err),
+				})
+			}
+
+			resolve.Invoke(map[string]interface{}{
+				"result": fmt.Sprintf("%s", err),
+			})
+		}()
+
+		return nil
+	})
+
+	promiseConstructor := js.Global().Get("Promise")
+	return promiseConstructor.New(handler)
+}
+
+// // ReadPoolUnlock unlocks tokens in expired read pool
+func ReadPoolUnlock(this js.Value, p []js.Value) interface{} {
+	poolID := p[0].String()
+	fee, _ := strconv.ParseInt(p[1].String(), 10, 64)
+
+	handler := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		resolve := args[0]
+		reject := args[1]
+
+		go func() {
+			err := sdk.ReadPoolUnlock(poolID, fee)
+			if err != nil {
+				reject.Invoke(map[string]interface{}{
+					"error": fmt.Sprintf("ReadPoolUnlock failed. Reason: %s", err),
+				})
+			}
+
+			resolve.Invoke(map[string]interface{}{
+				"result": fmt.Sprintf("%s", err),
+			})
+		}()
+
+		return nil
+	})
+
+	promiseConstructor := js.Global().Get("Promise")
+	return promiseConstructor.New(handler)
+}
+
+// //
+// // stake pool
+// //
+
+// // GetStakePoolInfo for given client, or, if the given clientID is empty,
+// // for current client of the sdk.
+func ZBOXGetStakePoolInfo(this js.Value, p []js.Value) interface{} {
+	blobberID := p[0].String()
+	handler := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		resolve := args[0]
+		reject := args[1]
+
+		go func() {
+			result, err := sdk.GetStakePoolInfo(blobberID)
+			if err != nil {
+				reject.Invoke(map[string]interface{}{
+					"error": fmt.Sprintf("GetStakePoolInfo failed. Reason: %s", err),
+				})
+			}
+
+			stakePool, err := json.Marshal(result)
+			if err != nil {
+				reject.Invoke(map[string]interface{}{
+					"error": fmt.Sprintf("GetStakePoolInfo to JSON Failed. Reason: %s", err),
+				})
+			}
+
+			resolve.Invoke(map[string]interface{}{
+				"result": string(stakePool),
+			})
+		}()
+
+		return nil
+	})
+
+	promiseConstructor := js.Global().Get("Promise")
+	return promiseConstructor.New(handler)
+}
+
+// // GetStakePoolUserInfo obtains blobbers/validators delegate pools statistic
```

Comment:
Function header comments format is: // GetStakePoolUserInfo but not // // GetStakePoolUserInfo . Please refactore everywhere

Reply:
I remove all comment in the `wasm/sdk.go` because it is already in the `zboxcore/sdk/sdk.go`